Source/WebCore/html/HTMLMediaElement.cpp

```diff
@@ -2363,13 +2372,15 @@ bool HTMLMediaElement::isSafeToLoadURL(const URL& url, InvalidURLAction actionIf
         if (actionIfInvalid == Complain) {
             if (frame)
                 FrameLoader::reportBlockedLoadFailed(*frame, url);
-            ERROR_LOG(LOGIDENTIFIER, url , " was rejected because the port is not allowed");
+            if (shouldLog)
+                ERROR_LOG(LOGIDENTIFIER, url , " was rejected because the port is not allowed");
         }
         return false;
     }
 
     if (!isAllowedToLoadMediaURL(*this, url, isInUserAgentShadowTree())) {
-        ERROR_LOG(LOGIDENTIFIER, url, " was rejected by Content Security Policy");
+        if (shouldLog)
+            ERROR_LOG(LOGIDENTIFIER, url, " was rejected by Content Security Policy");
```

Comment:
Ditto

Reply:
I missed those comments and had started the merge already. Will do in a follow up fix